e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -0,0 +1,711 @@
+          ...TARGET_QUESTION.query,
+          filter: ["and", QUERY_FILTER_CREATED_AT, QUERY_FILTER_QUANTITY],
+        });
+      });
+    });
+
+    it("does not allow setting saved question as custom destination if user has no permissions to it", () => {
+      cy.createCollection({ name: RESTRICTED_COLLECTION_NAME }).then(
+        ({ body: restrictedCollection }) => {
+          cy.updateCollectionGraph({
+            [USER_GROUPS.COLLECTION_GROUP]: {
+              [restrictedCollection.id]: "none",
+            },
+          });
+
+          cy.createQuestion({
+            ...TARGET_QUESTION,
+            collection_id: restrictedCollection.id,
+          });
+        },
+      );
+
+      cy.signOut();
+      cy.signInAsNormalUser();
+
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Go to a custom destination").click();
+      cy.get("aside").findByText("Saved question").click();
+
+      modal().findByText(RESTRICTED_COLLECTION_NAME).should("not.exist");
+      modal().findByText(TARGET_QUESTION.name).should("not.exist");
+    });
+
+    it("allows setting URL as custom destination and changing it back to default click behavior", () => {
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Go to a custom destination").click();
+      cy.get("aside").findByText("URL").click();
+      modal().within(() => {
+        cy.findByRole("textbox").type(URL);
+        cy.button("Done").click();
+      });
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      onNextAnchorClick(anchor => {
+        expect(anchor).to.have.attr("href", URL);
+        expect(anchor).to.have.attr("rel", "noopener");
+        expect(anchor).to.have.attr("target", "_blank");
+      });
+      clickLineChartPoint();
+
+      cy.log("allows to change click behavior back to the default");
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").icon("close").first().click();
+      cy.get("aside")
+        .findByText("Open the Metabase drill-through menu")
+        .click();
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      clickLineChartPoint();
+      assertDrillThroughMenuOpen();
+    });
+
+    it("allows setting URL with parameters as custom destination", () => {
+      const urlWithParams = `${URL}{{${DASHBOARD_FILTER_TEXT.slug}}}/{{${COUNT_COLUMN_ID}}}/{{${CREATED_AT_COLUMN_ID}}}`;
+      const escapedUrlWithParams = escapeCypressCurlyBraces(urlWithParams);
```

Comment:
Can we use `{ parseSpecialCharSequences: false }` when typing instead of introducing this util?

Reply:
Nice, thanks! ceb6745f0c93f35efcf27f5feac34da90d607269